android-bindings/src/bindings.rs

```diff
@@ -1204,16 +1644,21 @@ pub unsafe extern "C" fn Java_com_mobilecoin_lib_TransactionBuilder_add_1input(
     view_private_key: JObject,
 ) {
     jni_ffi_call(&env, |env| {
+        android_logger::init_once(Config::default().with_min_level(Level::Trace));
+        error!("this is printed by default");
         let mut tx_builder: MutexGuard<TransactionBuilder<FogResolver>> =
             env.get_rust_field(obj, RUST_OBJ_FIELD)?;
-
         let ring: Vec<TxOut> = (0..env.get_array_length(ring)?)
             .map(|index| {
                 let obj = env.get_object_array_element(ring, index)?;
                 let tx_out: MutexGuard<TxOut> = env.get_rust_field(obj, RUST_OBJ_FIELD)?;
                 Ok(tx_out.clone())
             })
             .collect::<Result<_, jni::errors::Error>>()?;
+        //for tx_out in ring.iter() {
+        //debug!("[SAMDEALY][RUST] TxOut {}",
+        // hex::encode(mc_util_serial::encode(tx_out)));
+        //}
```

Comment:
Is this code needed? If so please convert the PR to Draft, if not please remove any old commented-out code.

Reply:
It looks like this might have been there for generating some data. I've removed the comments are this isn't part of the implementation.